trunk/mailer.smtp.class.php

```diff
@@ -38,6 +38,6 @@ public function configure_phpmailer($phpmailer) {
         $phpmailer->Username = 'SMTP_Injection';
         $phpmailer->Password = $options['password'];
 
-        $phpmailer->set('X-MSYS-API', json_encode($x_msys_api));
+        $phpmailer->addCustomHeader('X-MSYS-API', json_encode($x_msys_api));
```

Comment:
I don't know why `set` does not work (http://phpmailer.worxware.com/?pg=methods), but it looks like `addCustomHeader()` should do `addCustomHeader('X-MSYS-API: '.json_encode())` from the docs